site/content/en/docs/installation/_index.md

```diff
@@ -66,6 +66,8 @@ to scrape metrics from kueue components, run the following command:
 kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/prometheus.yaml
 ```
 
+> **Note**: This command rely on CRD at `https://github.com/prometheus-operator/kube-prometheus/blob/main/manifests/setup/0servicemonitorCustomResourceDefinition.yaml`, please ensure that CRD is installed first.
+
```

Comment:
This feature completely depends on the Prometheus. 
So, instead of this Note, could you mention that we need to set up prometheus based on https://prometheus-operator.dev/.